packages/core/src/Stores/client-store.js

```diff
@@ -295,6 +295,46 @@ export default class ClientStore extends BaseStore {
         );
     }
 
+    @computed
+    get is_europe() {
```

Comment:
should we rename it to is_eu? because there are countries in europe that are not part of eu and we don't need to regulate